### PR TITLE
added tests for models

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
@AlexanderDokuchaev

**Comment by AlexanderDokuchaev** [comment]
run tests

**Comment by AlexanderDokuchaev** [comment]
run tests

**Comment by AlexanderDokuchaev** [comment]
run tests

**Comment by AlexanderDokuchaev** [comment]
test models

**Comment by AlexanderDokuchaev** [comment]
run tests

File: tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                                   item['id'] in selected_indexes]
+
+        with open(path, 'w') as write_file:
+            json.dump(content, write_file)
+
+    @classmethod
+    def setUpClass(cls):
+        os.makedirs(cls.coco_dir, exist_ok=True)
+        if not os.path.exists(os.path.join(cls.coco_dir, 'val2017.zip')):
+            os.system(f'wget http://images.cocodataset.org/zips/val2017.zip -P {cls.coco_dir}')
```
Comment (AlexanderDokuchaev) [inline tests/test_models.py]: Please add '--no-verbose' to disable progress bar.

**Comment by AlexanderDokuchaev** [comment]
run tests

File: tests/test_models.py (hunk 1 above)
Comment (Ilya-Krylov) [reply tests/test_models.py]: added

**Comment by AlexanderDokuchaev** [comment]
run tests

**Comment by AlexanderDokuchaev** [comment]
run tests

**Comment by AlexanderDokuchaev** [comment]
run tests

**Comment by AlexanderDokuchaev** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
run tests

**Pull request merged**